### ISSUE TITLE
UART Cubieboard

### ISSUE THREAD

**Issue by jarain78** [cubieplayer/Cubian#409]
Hi, this is other questions. Is it posible to configure other UART-Port on cubieboard using cubian?.

Thanks 

jarain78

**Comment by danfos** [comment]
Yes, that is possible although it is nothing Cubian specific, see:

https://www.domotiga.nl/projects/domotiga/wiki/Hardware_Cubieboard2#Enabling-an-additional-UART

and the links given there.

**Comment by jarain78** [comment]
Hi, thanks for your answer, but I have other question. The link that you give me is for A20, but I have the A10 cubieboard is the same process. This is my cubieboard.fex:

uart_port = 4
uart_type = 2
uart_tx = port:PH04<4><1><default><default>
uart_rx = port:PH05<4><1><default><default>

[uart_para5]
uart_used = 0
uart_port = 5
uart_type = 2
uart_tx = port:PH06<4><1><default><default>
uart_rx = port:PH07<4><1><default><default>

[uart_para6]
uart_used = 0
uart_port = 6
uart_type = 2
uart_tx = port:PA12<4><1><default><default>
uart_rx = port:PA13<4><1><default><default>

But I can't find the pins PH06 and PH07. 

http://abhinavgupta2812.files.wordpress.com/2013/08/gpio_defination_large.jpg

http://linux-sunxi.org/A10/PIO#PH06

And use this expansion board :

http://cubieboard.org/2013/11/30/dvk521-cubieboard12-baseboard-available-now/

Thank for your help.

jarain78

**Comment by danfos** [comment]
I think you are better off with this question at http://www.cubieforums.com/index.php/board,6.0.html
